spark-3.1-spanner-lib/src/main/java/com/google/cloud/spark/spanner/SpannerRDD.java

```diff
@@ -14,4 +14,56 @@
 
 package com.google.cloud.spark.spanner;
 
-public class SpannerRDD {}
+import com.google.cloud.spanner.BatchClient;
+import com.google.cloud.spanner.BatchReadOnlyTransaction;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.TimestampBound;
+import org.apache.spark.Dependency;
+import org.apache.spark.InterruptibleIterator;
+import org.apache.spark.Partition;
+import org.apache.spark.SparkContext;
+import org.apache.spark.TaskContext;
+import org.apache.spark.rdd.RDD;
+import org.apache.spark.sql.catalyst.InternalRow;
+import scala.collection.Seq;
+import scala.collection.Seq$;
+
+public class SpannerRDD extends RDD<InternalRow> {
```

Comment:
Why is the SpannerRDD needed? Data Source v2 does not require any RDD implementation (unlike the v1)

Reply:
Yeah I was implementing both dialects concurrently but no more.